---a/net/minecraft/command/WrongUsageException.java
+++b/net/minecraft/command/WrongUsageException.java
@@ -1,12 +1,11 @@
 package net.minecraft.command;
 
-public class WrongUsageException extends SyntaxErrorException
-{
+public class WrongUsageException extends SyntaxErrorException {
+
     private static final String __OBFID = "CL_00001192";
 
-    public WrongUsageException(String message, Object ... replacements)
-    {
+    public WrongUsageException(String message, Object... replacements) {
         super(message, replacements);
     }
-}
+}
